pages.es/osx/rargs.md

```diff
@@ -0,0 +1,25 @@
+# rargs
+
+> Ejecuta un comando por cada línea de entrada estándar.
+> Como `xargs`, pero con soporte de coincidencia de patrones.
+> Más información: <https://github.com/lotabout/rargs>.
+
+- Ejecuta un comando por cada línea de entrada, como `xargs` (`{0}` indica dónde sustituir en el texto):
+
+`{{command}} | rargs {{command}} {0}`
+
+- Realiza una ejecución en seco, lo cual imprime los comandos que se ejecutarían en lugar de ejecutarlos:
+
+`{{command}} | rargs -e {{command}} {0}`
+
+- Elimina la extensión `.bak` de todos los archivos de una lista:
+
+`{{command}} | rargs -p '(.*).bak mv {0} {1}`
+
+- Ejecuta comandos en paralelo:
+
+`{{command}} | rargs -w {{max-procs}}`
+
+- Considera cada línea de entrada separada por un carácter NUL (`\0`) en lugar de una nueva línea (`\n`):
```

Comment:
```suggestion
- Lee líneas de entrada separadas por el caracter `NUL` (`\0`) en vez de `LF` (`\n`):
```

I believe the abbreviations of ASCII characters should be wrapped between backticks because they are technical terms.

I do not know if the newline character can be referred to as `nueva línea`, so to prevent ambiguities I would use `LF`.